safehttp/plugins/csp/csp.go

```diff
@@ -167,6 +169,12 @@ func (f FramingPolicy) Serialize(nonce string) string {
 	return b.String()
 }
 
+// toXFO returns a value of the X-Frame-Options header that reflects settings of the policy.
+func (f FramingPolicy) toXFO() string {
```

Comment:
nit: I'd name it "XFrameOptions".

The "to" or "get" prefixes in Go are quite uncommon.

Reply:
Done.